src/components/SchemaInput.vue

```diff
@@ -78,8 +78,9 @@
   })
 
   const schemaDefaultValues = getSchemaDefaultValues(props.schema)
-  const defaultValues = merge({}, schemaDefaultValues, props.modelValue ?? {})
-  const unmappedDefaultValues = mapper.map('SchemaValues', { values: defaultValues, schema: props.schema }, 'SchemaValuesRequest')
+  const unmappedModelValue = mapper.map('SchemaValues', { values: props.modelValue ?? {}, schema: props.schema }, 'SchemaValuesRequest')
+  const unmappedDefaultValues = merge({}, schemaDefaultValues, unmappedModelValue)
```

Comment:
I believe `getSchemaDefaultValues` returns `SchemaValues` but you're merging them with `SchemaValuesRequest`. I think you probably want to merge them with the already mapped modalValue